src/test/unit_tests/ci/game_utils.test.ts

```diff
@@ -327,23 +327,23 @@ describe("game utils", () => {
                         });
                     });
 
-                    describe("Collab should fail", () => {
-                        it("should not match any group", async () => {
-                            State.aliases.artist = {};
-                            const nonMatchArtist = "PSY + Hyuna";
+                    describe("Collab should match", () => {
+                        it("should match group", async () => {
+                            const artistName = "PSY + Hyuna";
+
                             const matchResults = await getMatchingGroupNames(
                                 State.aliases.artist,
-                                [nonMatchArtist],
+                                [artistName],
                             );
 
                             assert.deepStrictEqual(
```

Comment:
Can you add an assert that checks that `addedByUser` is true? And another test that checks that its false if it was added through an artist instead?